extensions/amp-fit-text/0.1/amp-fit-text.js

```diff
@@ -41,6 +43,9 @@ class AmpFitText extends AMP.BaseElement {
     /** @private {number} */
     this.maxFontSize_ = -1;
 
+    /** @private {?unlistenDef} */
```

Comment:
Should this be `function`? Where is `unlistenDef` defined?

Reply:
UnlistenDef is defined in amp.extern.js (https://github.com/ampproject/amphtml/blob/master/build-system/externs/amp.extern.js#L414-L418) and refers to a function which specifically removes a listener, in this case the resizeObserver. I think its just a shade more precise than the `function` type declaration for this usage.

Reply:
Oh I see - then should this be capitalized?
```suggestion
    /** @private {?UnlistenDef} */
```